api/server.js

```diff
@@ -0,0 +1,29 @@
+const express = require("express");
+
+const server = express();
+
+server.use(express.json());
+
+const games = [];
+
+//POST Endpoint
+server.post("/games", (req, res) => {
+  const { title, genre, releaseYear } = req.body;
+
+  if (!title || !genre || !releaseYear) {
+    return res
+      .status(422)
+      .json({ error: "Please provide a title, genre, and release year." });
```

Comment:
Good data validation and error message specifying exactly what data the endpoint expects.